XSDElement/src/main/java/com/modelsolv/reprezen/gentemplates/xsdelement/xtend/PropertyRealizationHelper.xtend

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright © 2013, 2016 Modelsolv, Inc.
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains the property
+ * of ModelSolv, Inc. See the file license.html in the root directory of
+ * this project for further information.
+ *******************************************************************************/
+
+package com.modelsolv.reprezen.gentemplates.xsdelement.xtend
+
+import com.google.common.collect.Iterables
+import com.modelsolv.reprezen.restapi.PropertyRealization
+import com.modelsolv.reprezen.restapi.ServiceDataResource
+import com.modelsolv.reprezen.restapi.TypedMessage
+import com.modelsolv.reprezen.restapi.datatypes.PrimitiveProperty
+import com.modelsolv.reprezen.restapi.datatypes.ReferenceProperty
+
+class PropertyRealizationHelper {
+	
+	extension FeatureHelper = new FeatureHelper
+
+	def private dispatch Iterable<PropertyRealization> getSequencePropRzs(ServiceDataResource dataResource) {
+		dataResource.includedProperties.getSequencePropRzs
+	}
+
```

Comment:
Since all the `getSequencePropRzs` methods are private, we can be certain that they're only called from the corresponding `hasSequencePropRzs` methods in this same class. In all cases, the argument types exactly match, so I don't think this needs to be dispatch. I also think it'd be clearer for these to be at the bottom, near the public dispatch methods that use them.

On the other hand, if `getSequencePropRzs` is likely to be a useful method in its own right for this set of utility classes, even if it's not used currently, then perhaps make the methods private and leave them as dispatch methods.